libcst/codemod/visitors/tests/test_remove_imports.py

```diff
@@ -377,6 +377,46 @@ def foo() -> None:
             context_override=CodemodContext(full_module_name="a.b.foobar"),
         )
 
+    def test_remove_dotted_import(self) -> None:
+        before = """
+            import a.b, a.b.c
+            import e.f
+            import g.h
+            import x.y, x.y.z
+
+            def foo() -> None:
+                a.b
+                e.g
+                g.h.i
+                x.y.z
+        """
+
+        after = """
+            import a.b, a.b.c
+            import e.f
+            import g.h
+            import x.y.z
+
+            def foo() -> None:
+                a.b
+                e.g
+                g.h.i
+                x.y.z
+        """
+
+        self.assertCodemod(
+            before,
+            after,
+            [
+                ("a.b", None, None),
+                ("a.b.c", None, None),
+                ("e.f", None, None),
+                ("g.h", None, None),
+                ("x.y", None, None),
+                ("x.y.z", None, None),
+            ],
+        )
```

Comment:
If you add this as a test case of `RemoveUnusedImportsCommandTest`, you don't need to manually iterate all the imports.
https://github.com/Instagram/LibCST/blob/master/libcst/codemod/commands/tests/test_remove_unused_imports.py#L12-L18

Reply:
Good point, I haven't thought about that :)